metrics/xrootd_metrics.go

```diff
@@ -1675,3 +1660,32 @@ func handleOSSPacket(blobs [][]byte) error {
 
 	return nil
 }
+
+func handleThrottlePacket(blobs [][]byte) error {
```

Comment:
Can you leave a docstring that explains how the function handles the packet and any assumptions that must be valid for it to do its job?